apps/talon/talon_debug_window/talon_debug_window.talon

```diff
@@ -0,0 +1,24 @@
+# this functionality is only available in the talon beta
+# note: these commands are only useful when the search box is focused
+app: talon_debug_window
+-
+# uncomment user.talon_populate_lists tag to activate talon-specific lists of actions, scopes, modes etcetera.
+# Do not enable this tag with dragon, as it will be unusable.
+# with conformer, the latency increase may also be unacceptable depending on your cpu
+# see https://github.com/talonhub/community/issues/600
+# tag(): user.talon_populate_lists
+
+tag {user.talon_tags}: "{talon_tags}"
+
+#commands for dictating key combos
```

Comment:
Would be helpful if there is a comment that explains how to use these in the context of the Talon debug window.